Resources/Locale/en-US/_NF/ghost/roles/ghost-role-component.ftl

```diff
@@ -36,3 +36,13 @@ ghost-role-information-taxibot-description = Drive passengers to where they need
 
 ghost-role-information-hovertaxibot-name = Hovertaxibot
 ghost-role-information-hovertaxibot-description = Fly passengers to where they need to go, remember to check they can breathe!
+
+ghost-role-information-aberrant-flesh-horror-dungeon-name = Aberrant Flesh Horror
+ghost-role-information-aberrant-flesh-horror-dungeon-description = Defend the expedition dungeon with other flesh monstrosities!
+ghost-role-information-aberrant-flesh-horror-dungeon-rules = You are a [color=red][bold]Team Antagonist[/bold][/color] with all dungeon mobs. Your goal is to seek confrontation with players on dungeon grounds.
+                                        Please [color=yellow]note[/color] that the all usual server rules are still applicable. Additionally:
+                                        - [color=red]DO NOT[/color] grief players by intentionally damaging the player's shuttle or any machinery inside.
+                                        - [color=red]DO NOT[/color] grief players by hiding from them until the expedition timer runs out.
+                                        - [color=red]DO NOT[/color] grief players by gibbing them.
+                                        - [color=red]DO NOT[/color] grief players by intentionally destroying or hiding valuable loot in the dungeon.
+                                        - [color=red]DO NOT[/color] leave the planet.
```

Comment:
Interesting, should look into getting something like this into the xeno queen/space dragon mobs.  Will see how it looks.

Reply:
It looks kinda fine, but perhaps this text suffers from the same issues any rules usually suffer: it might be defined too clearly which might be read by certain players as invitation to look for a way to grief without "technically" breaking any role rules. Idk. In any case, it's game admin issue.

Reply:
Minor rewrite, now fits on a window (at least with my UI scaling):

![image](https://github.com/user-attachments/assets/df8a8f57-7d97-4a4c-a6b5-b92392ffed42)